projects/organization-management/src/app/models/organization/organization.mapper.ts

```diff
@@ -0,0 +1,21 @@
+import { OrganizationDocument } from './organization.interface';
+import { Organization } from './organization.model';
+
+export class OrganizationMapper {
+  static fromData(payload: OrganizationDocument): Organization {
+    const data = payload.data;
+
+    if (!data) {
+      throw new Error(`'organizationDocument' is required`);
+    }
+
+    return {
+      id: data.id,
+      name: data.attributes ? data.attributes.name : undefined,
```

Comment:
Please use safe navigation with `?.` operator

Reply:
ALso `id` and `name` are both mandatory attributes according to the API description